### DISCUSSION TITLE
Attribute error with scipy.stats

### DISCUSSION THREAD

**Discussion by boborodono** [sfu-db/dataprep#859] [Q&A]
Anyone ever get this error and know how to handle it. Couldn't find anything online. Imported stats from scipy and tried differen versions but same error. It also says there is an error with a specific column but it's always the first column no matter the dataset and when I drop the problematic column the error is then attributed to the next column. Any ideas?

[AttributeError: scipy.stats.stats is deprecated and has no attribute _power_div_lambda_names. Try looking in scipy.stats instead.]()

**Comment by jinglinpeng** [comment]
Hi @boborodono , thanks for proposing this question. This is the scipy version problem. A related issue: https://github.com/sfu-db/dataprep/issues/840. Can you try to set it to 1.5.4 or 1.7.1? We'll fix it in the next release.
